process_bigraph/tests.py

```diff
@@ -44,9 +50,11 @@ def test_merge_collections():
 def test_process():
     process = IncreaseProcess({'rate': 0.2})
     schema = process.schema()
-    state = types.fill(schema)
+    state = types.fill(schema['inputs'])
+    state = types.fill(schema['outputs'])
```

Comment:
maybe we should have a `types.fill_from_schema` method that knows about inputs and outputs?

Reply:
In a way `inputs` and `outputs` each define their own schema.... we have other schemas (notably the `config_schema` for processes) which are singular, so we need to support both. Happy to add a helper function for this.